app/views/users/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ ] In order to be consistent with the layout of the wireframe, please introduce a padding around the form.
- [ ] Also, notice that in the wireframe, there are no labels. Instead set the placeholders for each input element so that you have a design that looks more like the wireframe design.
The above requested changes are project requirements

## Wireframe
![image](https://github.com/Outkarm/blog-app/assets/46839548/71eeaf60-268c-40b9-a478-3f5238aeb5fb)

## Your Implementation
![image](https://github.com/Outkarm/blog-app/assets/46839548/ca7b7d7c-2a36-42d6-8808-737002248da9)